pkg/kn/commands/service_create.go

```diff
@@ -73,7 +73,7 @@ func NewServiceCreateCommand(p *KnParams) *cobra.Command {
 			if err != nil {
 				return err
 			}
-
+			fmt.Fprintf(cmd.OutOrStdout(), "Service %s is created in %s namespace.\n", args[0], namespace)
```

Comment:
i would recommend adding single quotes around names. also consistently putting name after resource type. something like this:

```
Service '%s' was successfully created in namespace '%s'.\n
```

Reply:
Ok, updated the PR, also updated the info message for `service delete` too.